PROJECT-CLEARWATER-MIB

```diff
@@ -2447,4 +2450,237 @@ mementoGroup OBJECT-GROUP
     STATUS      current
     DESCRIPTION "Statistics exposed by Memento"
     ::= { projectClearwaterObjectGroups 5 }
+
+-- Astaire Entries
+
+astaireBucketsNeedingResync OBJECT-TYPE
+    SYNTAX      Unsigned32
+    MAX-ACCESS  read-only
+    STATUS      current
+    DESCRIPTION "Count of buckets needing to be synchronized for the current
+                 resynchronization operation."
+    ::= { astaire 1 }
+
+astaireBucketsResynchronized OBJECT-TYPE
+    SYNTAX      Unsigned32
+    MAX-ACCESS  read-only
+    STATUS      current
+    DESCRIPTION "Number of buckets already resynchronized during the current
+                 resynchronization operation."
+    ::= { astaire 2 }

```

Comment:
If there's no ongoing synchronization, there is no resynchronization operation, and these return 0.  I've updated the definition.